[2022-03-17 18:20:55,168] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_example_dag.task_group_Dimension_table.load_users_dim_table manual__2022-03-17T10:17:55.530886+00:00 [queued]>
[2022-03-17 18:20:55,180] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_example_dag.task_group_Dimension_table.load_users_dim_table manual__2022-03-17T10:17:55.530886+00:00 [queued]>
[2022-03-17 18:20:55,180] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 18:20:55,181] {taskinstance.py:1244} INFO - Starting attempt 1 of 3
[2022-03-17 18:20:55,181] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 18:20:55,190] {taskinstance.py:1264} INFO - Executing <Task(LoadDimensionOperator): task_group_Dimension_table.load_users_dim_table> on 2022-03-17 10:17:55.530886+00:00
[2022-03-17 18:20:55,192] {standard_task_runner.py:52} INFO - Started process 26881 to run task
[2022-03-17 18:20:55,198] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'udac_example_dag', 'task_group_Dimension_table.load_users_dim_table', 'manual__2022-03-17T10:17:55.530886+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/udac_example_dag.py', '--cfg-path', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmpvjs05eaq', '--error-file', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmpihcwuaje']
[2022-03-17 18:20:55,199] {standard_task_runner.py:77} INFO - Job 148: Subtask task_group_Dimension_table.load_users_dim_table
[2022-03-17 18:20:55,287] {logging_mixin.py:109} INFO - Running <TaskInstance: udac_example_dag.task_group_Dimension_table.load_users_dim_table manual__2022-03-17T10:17:55.530886+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2022-03-17 18:20:55,393] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=udacity
AIRFLOW_CTX_DAG_ID=udac_example_dag
AIRFLOW_CTX_TASK_ID=task_group_Dimension_table.load_users_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T10:17:55.530886+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-17T10:17:55.530886+00:00
[2022-03-17 18:20:55,394] {load_dimension.py:83} INFO - Starting to truncate users
[2022-03-17 18:20:55,411] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-17 18:20:56,750] {dbapi.py:225} INFO - Running statement: 
        TRUNCATE TABLE "public"."users"
        , parameters: None
[2022-03-17 18:20:57,376] {load_dimension.py:93} INFO - Starting to insert users
[2022-03-17 18:20:57,379] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-17 18:20:57,380] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-03-17 18:20:57,381] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-03-17 18:20:57,382] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-03-17 18:20:57,382] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:20:57,383] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:20:57,384] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:20:57,385] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-03-17 18:20:57,386] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:20:57,387] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:20:57,388] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:20:57,389] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:20:57,398] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-17 18:20:58,636] {dbapi.py:225} INFO - Running statement: 
        INSERT INTO "public"."users" (userid, first_name, last_name, gender,level)
 
        SELECT distinct userid, firstname, lastname, gender, level
        FROM staging_events
        WHERE page='NextSong'
    
        , parameters: None
[2022-03-17 18:20:59,059] {dbapi.py:233} INFO - Rows affected: 104
[2022-03-17 18:20:59,307] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=udac_example_dag, task_id=task_group_Dimension_table.load_users_dim_table, execution_date=20220317T101755, start_date=20220317T102055, end_date=20220317T102059
[2022-03-17 18:20:59,340] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-17 18:20:59,375] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
